index.html

```diff
@@ -0,0 +1,158 @@
+</head>
+
+<body>
+    <header>
+        <nav class="top-navbar">
+            <ul class="top-nav-items">
+                <li><a href="#"><img src="./assets/icons/facebook.png" alt="facebook"></a></li>
+                <li><a href="#"><img src="./assets/icons/twitter.png" alt="twitter"></a></li>
+                <li><a href="#" class="link-top-nav">English</a></li>
+                <li><a href="#" class="link-top-nav">My Page</a></li>
+                <li><a href="#" class="link-top-nav">Logout</a></li>
+            </ul>
+        </nav>
+        <nav class="navbar">
+            <div class="nav-img-container">
+            <a href="#" class="navbar-img-logo"><img src="./assets/images/logo.png" alt="Saarc Logo"></a>
+            </div>
+            <div class="hamburger">
+                <span class="bar"></span>
+                <span class="bar"></span>
+                <span class="bar"></span>
+            </div>
+
+            <ul class="nav-menu">
+                <li class="nav-item">
+                    <a href="./aboutme.html" class="nav-link">About</a>
+                </li>
+                <li class="nav-item">
+                    <a href="#" class="nav-link">Program</a>
+                </li>
+                <li class="nav-item">
+                    <a href="#" class="nav-link">Join</a>
+                </li>
+                <li class="nav-item">
+                    <a href="#" class="nav-link">Sponser</a>
+                </li>
+                <li class="nav-item">
+                    <a href="#" class="nav-button">Campaign</a>
+                </li>
+            </ul>
+            
+        </nav>
+    </header>
+    <main>
+        <!--First Section-->
+        <section class="first-section">
+            <div>
+            <p class="text-1">Dear Viewer's Welcome to the</p>
+            <h1 class="main-text">The 20th SAARC Summit</h1>
+            </div>
+            <div class="text-2">The South Asian Association for Regional Cooperation (SAARC) is the regional intergovernmental organization and geopolitical union of states in South Asia.</div>
+            <div class="location-container">
+            <h3 class="date">2022.05.17 ~ 18</h3>
+            <p class="location">@ Islamabad Capital of Pakistan.</p>
+            </div>
+        </section>
+        <!--Second Section-->
+        <section class="second-section">
+            <h2 class="main-p-text">Main Program</h2>
+            <span class="bar-gold"></span>
+            <div class="second-part">
+            <div class="parts">
+                <div class="parts-img-con">
+                <img src="./assets/icons/program_icon_01.png" alt="" class="parts-photo">
+                </div>
+                
+                <h3 class="activity-bar">Speech</h3>
+                <p>Listen to Leaders of the SAARC countries talking about ways to find solution to the economic growth, welfare and cultural development.</p>
+                
+            </div>
```

Comment:
- [ ] Each individual card should have an enclosing `<article>` element tag instead of meaningless `<div>` tag. This is according to semantic HTML. Please implement this 🙏 
- [ ] Please ensure that the "Speech" text uses the specified `#ec5242` color 🎨